nerfstudio/scripts/process_data.py

```diff
@@ -69,7 +65,7 @@ def main(self) -> None:
 
         summary_log = []
 
-        record3d_image_dir = self.data / "rgb"
+        record3d_image_dir = self.data[0] / "rgb"
```

Comment:
This feels weirdly necessary because of how we are typing data. Maybe there is a way around this?

Reply:
Yea, it is slightly annoying... Instead of having data be a Tuple of train and eval paths, we could keep data as is, and create a new argument eval_data. Let me try this.